pyproject.toml

```diff
@@ -48,3 +48,27 @@ indent_size = 2
 
 [tool.djlint.js]
 indent_size = 2
+
+[project]
+name = "cookiecutter-django"
+version = "2024.09.02"
+requires-python = "== 3.12"
+description = "A Cookiecutter template for creating production-ready Django projects quickly."
+readme = "README.md"
+license = { text = "BSD" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Framework :: Django :: 5.0",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Software Development",
+]
+keywords = ["cookiecutter", "Python", "projects", "project templates", " django", "skeleton", "scaffolding", "project directory", "setup.py"]
```

Comment:
```suggestion
keywords = ["cookiecutter", "Python", "projects", "project templates", " django", "skeleton", "scaffolding", "project directory"]
```

Reply:
If we decide to change the keywords I would opt for removing Python, projects, and project directory to.

Reply:
Fair point, yes 👍🏻